Doc/library/datetime.rst

```diff
@@ -513,9 +517,7 @@ Notes:
    :const:`MINYEAR` or larger than :const:`MAXYEAR`.
 
 (2)
-   This isn't quite equivalent to date1 + (-timedelta), because -timedelta in
-   isolation can overflow in cases where date1 - timedelta does not.
-   ``timedelta.seconds`` and ``timedelta.microseconds`` are ignored.
+   timedelta.seconds and timedelta.microseconds are ignored.
```

Comment:
Just a note that has changed from:

```
`timedelta.seconds` and `timedelta.microseconds` are ignored
```

In the original they were in code blocks, but the documentation doesn't seem terribly consistent on this point. We should probably pick one and stick with it.

A cursory look at the rest of the documentation seems to indicate that "`timedelta.X`" is more common than "timedelta.X", but not by much, and often in expressions.